### PR TITLE
Update readme with new descriptive text

### PR DESCRIPTION
**Pull request by sallain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
+Enduro is a new application under development by Artefactual Systems. Originally
+created as a more stable replacement for Archivematica’s automation tools
+library of scripts, it has since evolved into a flexible tool to be paired with
+preservation applications to provide initial ingest activities such as content
+and structure validation, packaging, and more.
 
-It's a **proof of concept** at its very early stages. It aims to cover our
-client's needs while exploring new and innovative ways to build durable and
-fault-tolerant workflows suited for preservation.
+Enduro aims to cover our client's needs while exploring new and innovative ways
```
Comment (fiver-watson) [inline README.md]: nitpick - should perhaps be `clients'` needs, to make plural clients possessive? Or, `our client needs` (drop the s entirely)?

**Review by fiver-watson — APPROVED** [review]
That first paragraph is truly brilliant, stuff of genius even :grin: 

One minor nitpick noted for your consideration; otherwise LGTM!

File: README.md (hunk 1 above)
Comment (sallain) [reply README.md]: Thanks! That's legacy text, so may have been from the time when there was literally only one client :joy:

**Review by fiver-watson — APPROVED** [review]
SHIP IT

**Pull request merged**